lib/components/cards/CardHeader.dart

```diff
@@ -13,7 +13,7 @@ class CardHeader extends StatelessWidget {
     return Row(
       mainAxisAlignment: MainAxisAlignment.spaceBetween,
       children: [
-        Flexible(
+        Expanded(
```

Comment:
Expanded is a Flexible subclass. Since this was the only Flexible, and Expanded is used elsewhere, I swapped it out for consistency.